src/features/administrative/scripts/import-statistics.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import chalk from 'chalk'
+import { getFromFhir, sendToFhir } from '../../utils'
+import { getStatistics, LocationStatistic } from './utils'
+
+async function getLocationByIdentifier(
+  identifier: string
+): Promise<fhir.Location> {
+  const locationSearchResult = await getFromFhir(
+    `/Location/?identifier=${identifier}`
+  )
+  const location: fhir.Location =
+    locationSearchResult &&
+    locationSearchResult.entry &&
+    locationSearchResult.entry[0] &&
+    locationSearchResult.entry[0].resource
+  return location
+}
+
+function generateStatisticalExtensions(sourceStatistic: LocationStatistic) {
+  const malePopulations = []
+  const femalePopulations = []
+  const totalPopulations = []
+  const birthRates = []
+
+  for (const year of sourceStatistic.years) {
+    femalePopulations.push({
+      [year.year]: year.female_population
+    })
+    malePopulations.push({
+      [year.year]: year.male_population
+    })
+    totalPopulations.push({
+      [year.year]: year.population
+    })
+    birthRates.push({
+      [year.year]: year.crude_birth_rate / 2
```

Comment:
This is not correct.  There should be a male/female ratio used.  I need to discuss with @eduffus what to do about this